templates/common/infra/terraform/core/host/appservice/appservicenode/appservicenode_output.tf

```diff
@@ -0,0 +1,8 @@
+output "APP_DEFAULT_HOST_NAME" {
+  value = azurerm_linux_web_app.web.default_hostname
+}
+
+output "APP_IDENTITY_PRINCIPAL_ID" {

```

Comment:
Do we need to prefix with APP?  I would prefer to be consistent with bicep if we can.

Reply:
renamed